FixtureGenerator/Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(name: "FixtureGenerator",
+                      platforms: [
+                          .macOS(.v10_15)
```

Comment:
I think this should be `.v12` because [the endpoint is available since 12.0](https://github.com/nerdishbynature/octokit.swift/pull/159/files?w=1#diff-eb990850863d128b29b6628bf4cfdaa9a3a03722d7db2aa019746a89b9ebba20R5) 🤔

Reply:
I was generally wondering if we could move all the methods to be available from 10.15 on because concurrency was backported, no?

Reply:
I agree and it will be correct if CI passes.